mytest/arith-threaded.k

```diff
@@ -0,0 +1,44 @@
+// Copyright (c) 2018 Runtime Verification, Inc. (RV-Match team). All Rights Reserved.
```

Comment:
this is entirely the wrong place for these tests. they should be in a new directory under k-distribution/tests/regression-new